docs/tokens.md

```diff
@@ -279,3 +279,289 @@
+| CheckboxDisabled | #CDD3DD | ![#CDD3DD](https://placehold.it/15/CDD3DD/000000?text=+) |
+| Radio | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| RadioChecked | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| RadioDisabled | #CDD3DD | ![#CDD3DD](https://placehold.it/15/CDD3DD/000000?text=+) |
+| RadioDisabledChecked | #A8B1BD | ![#A8B1BD](https://placehold.it/15/A8B1BD/000000?text=+) |
+## TextColor
+
+| Key | Value | Color |
+| --- | ----- | ----- |
+| Default | #121417 | ![#121417](https://placehold.it/15/121417/000000?text=+) |
+| Secondary | #49505A | ![#49505A](https://placehold.it/15/49505A/000000?text=+) |
+| Active | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| ActiveHover | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| Error | #DF320C | ![#DF320C](https://placehold.it/15/DF320C/000000?text=+) |
+| Disabled | #A8B1BD | ![#A8B1BD](https://placehold.it/15/A8B1BD/000000?text=+) |
+| InputLabel | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| InputPlaceholder | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| InputHelpText | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| InputErrorMessage | #DF320C | ![#DF320C](https://placehold.it/15/DF320C/000000?text=+) |
+| ButtonLink | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| ButtonLinkHover | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| ButtonPrimary | #FFFFFF | ![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) |
+| ButtonPrimaryDisabled | #A8B1BD | ![#A8B1BD](https://placehold.it/15/A8B1BD/000000?text=+) |
+| ButtonNeutral | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| ButtonNeutralHover | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| ButtonNeutralDisabled | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| ButtonTransparent | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| ButtonTransparentHover | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| ButtonTransparentDisabled | #6A7381 | ![#6A7381](https://placehold.it/15/6A7381/000000?text=+) |
+| ButtonDestructive | #FFFFFF | ![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) |
+| ButtonDestructiveDisabled | #A8B1BD | ![#A8B1BD](https://placehold.it/15/A8B1BD/000000?text=+) |
+| ButtonTransparentTertiary | #49505A | ![#49505A](https://placehold.it/15/49505A/000000?text=+) |
+| ButtonLinkTertiary | #49505A | ![#49505A](https://placehold.it/15/49505A/000000?text=+) |
+| TabActive | #2F73DA | ![#2F73DA](https://placehold.it/15/2F73DA/000000?text=+) |
+| LeftNavItemActive | #165ECC | ![#165ECC](https://placehold.it/15/165ECC/000000?text=+) |
+| Tooltip | #FFFFFF | ![#FFFFFF](https://placehold.it/15/FFFFFF/000000?text=+) |
```

Comment:
Can you actually update the script to delete the doc file first? It just appends to that file.

Reply:
I just pushed a change that I believe works. I'm not a node expert, but I did google it.